src/Network/Network/help/Add-AzApplicationGatewayBackendSetting.md

```diff
@@ -8,7 +8,7 @@ schema: 2.0.0
 # Add-AzApplicationGatewayBackendSetting
```

Comment:
Please also update the field `online version` in metadata. Here is an [example](https://github.com/Azure/azure-powershell/blob/52f66f5347f55f7791efa46c5784f843f3e116de/src/Network/Network/help/Add-AzApplicationGatewayIPConfiguration.md?plain=1#L5)

Reply:
I have updated the "online version"  in these help files